File: C/Exercises/4ª Lista de Exercícios (1S)/Exercício05.c
#include <stdio.h>

int main()
{
    int par = 0, impar = 0, num, i;
    int soma_par = 0, soma_impar = 0;
    float media_par, media_impar;
    printf("Digite o número desejado: \n");
    for (i = 0; i < 10; i++)
    {
        printf("-> ");
        scanf("%d", &num);
        
        if (num % 2 == 0)
        {
            par++;
            soma_par += num;
        }
        else 
        {
            impar++;
            soma_impar += num;
        }
    }
    
    media_impar = soma_impar / impar;
    media_par = soma_par / par;

    printf("A média dos números pares é %g.\n", media_par);
    printf("A média dos números ímpares é %g.\n", media_impar);

}

File: College/2ª Semestre/Programação de Computadores/3ª Lista de Exercícios (2S)/Exercício05.py
def primoOuNao(num):
    if num == 1:
        return False
    for i in range(2, num//+1):
        if num % i == 0:
            return False
    return True

n = int(input("Digite um número: "))

for i in range(1,n):
    if primoOuNao(i):
        print(i)
